src/flutter/flutter-wizard.ts

```diff
@@ -157,12 +157,14 @@ Set the ${chalk.cyan(
   clack.outro(`
     ${chalk.greenBright('Successfully installed the Sentry Flutter SDK!')}
     
-    ${chalk.cyan(
-      `You can validate your setup by launching your application and checking Sentry issues page afterwards
-    ${issuesPageLink}`,
-    )}
+    ${chalk.cyan('Next steps:')}
+    1. Run ${chalk.bold('flutter run')} to test the setup - we've added a test error that will trigger on app start
+    2. For production builds, run ${chalk.bold('flutter build ios --obfuscate --split-debug-info=build/debug-info')} (or android/macos) then ${chalk.bold('flutter pub run sentry_dart_plugin')} to upload debug symbols
```

Comment:
```suggestion
    2. For production builds, run ${chalk.bold('flutter build apk --obfuscate --split-debug-info=build/debug-info')} (or ios/macos) then ${chalk.bold('flutter pub run sentry_dart_plugin')} to upload debug symbols
```

Reply:
qq: do we need to make it clear that (apk | ios | macos) are not the only builds that we support?

for example on android it's possible to do `flutter build appbundle` or `flutter build apk` and on ios/macos there are a bunch of other ways as well. 

essentially it doesn't matter what type of production build they do, what's important is that they add these flags `--obfuscate --split-debug-info=build/debug-info'`

Reply:
i updated it as you suggested, generally users can see the docs. I want to create a ticket to better handle dSYM for android and iOS. For now they are at least given some instructions and links to docs